aion/web/elm/src/Update.elm

```diff
@@ -70,10 +72,14 @@ update msg model =
             case Decode.decodeValue answerFeedbackDecoder rawFeedback of
                 Ok answerFeedback ->
                     let
-                        x =
-                            Debug.log "feedback" answerFeedback.feedback
+                        answerToast =
+                            case answerFeedback.feedback of
+                                "incorrect" -> incorrectAnswerToast
+                                "close" -> closeAnswerToast
+                                "correct" -> correctAnswerToast
+                                _ -> (\x -> x)
```

Comment:
We will probably remodel the evaluation part in the future, so we could just go with Debug.crash here or at least log an error here.